### ISSUE TITLE
iOS: Preference for size of image thumbnails in story titles

### ISSUE THREAD

**Issue by samuelclay** [samuelclay/NewsBlur#1142]
After seeing Brett Simmons' new UI and thinking about what he got right, I think that the image previews next to story titles should have a third setting (apart from on/off) which is "Small thumbnail", which insets a small preview instead of taking up top to bottom space (and thereby shortening the title excessively, esp. on an iPad).
<img width="381" alt="timeline_light" src="https://user-images.githubusercontent.com/44229/47966205-e37b9280-e01d-11e8-95ec-693563f5af50.png">

**Comment by Dejal** [comment]
He's just showing the favicon-type image instead of an actual thumbnail as in NewsBlur. But you'd want to stick with the current layout, but make the thumbnail smaller, thus allowing more space for the title & preview text?

**Comment by samuelclay** [comment]
You're right, it's the favicon, but I just want to have a preference for small thumbnails so less of the title is truncated.
